.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
+profile=google
```

Comment:
I guess this should be in setup.cfg too?

Reply:
Or rather, we should just copy the settings we need from the google profile to keep it explicit

```
"force_single_line": True,
"force_sort_within_sections": True,
"lexicographical": True,
"single_line_exclusions": ("typing",),
```
([Taken from here](https://timothycrosley.github.io/isort/reference/isort/profiles/))

Reply:
Right, removed `.isort.cfg` and explicitly mentioned these params in `setup.cfg`.